x-pack/plugins/cases/public/components/connectors/connectors_registry.ts

```diff
@@ -28,17 +45,9 @@ export const createCaseConnectorsRegistry = (): CaseConnectorsRegistry => {
       connectors.set(connector.id, connector);
     },
     get: <UIProps>(id: string): CaseConnector<UIProps> => {
-      if (!connectors.has(id)) {
-        throw new Error(
-          i18n.translate('xpack.cases.connecors.get.missingCaseConnectorErrorMessage', {
-            defaultMessage: 'Object type "{id}" is not registered.',
-            values: {
-              id,
-            },
-          })
-        );
-      }
-      return connectors.get(id)!;
+      const connector = connectors.get(id);
+      assertConnectorExists(connector, id);
```

Comment:
Move the logic to an assertion so I can return the connector without having to put `!` at the end of the return statement.